packages/legacy/core/App/navigators/OnboardingStack.tsx

```diff
@@ -16,35 +16,35 @@ import PINCreate from '../screens/PINCreate'
 import PINEnter from '../screens/PINEnter'
 import PushNotification from '../screens/PushNotification'
 import { AuthenticateStackParams, Screens } from '../types/navigators'
+import { State } from '../types/state'
+import { Config } from '../types/config'
 
 import { useDefaultStackOptions } from './defaultStackOptions'
-
-type ScreenOptions = RouteConfig<
-  ParamListBase,
-  Screens,
-  StackNavigationState<ParamListBase>,
-  StackNavigationOptions,
-  StackNavigationEventMap
->
+import { DeviceEventEmitter } from 'react-native'
+import { getOnboardingScreens } from './OnboardingScreens'
+import { useOnboardingState } from '../hooks/useOnboardingState'
 
 const OnboardingStack: React.FC = () => {
-  const [, dispatch] = useStore()
+  const [store, dispatch] = useStore<State>()
   const { t } = useTranslation()
   const Stack = createStackNavigator()
   const theme = useTheme()
   const OnboardingTheme = theme.OnboardingTheme
   const carousel = createCarouselStyle(OnboardingTheme)
   const [
-    splash,
+    config,
+    Splash,
     pages,
     useBiometry,
```

Comment:
Since this is a screen (JSX) and not a hook it should probably be capitalized. Not something we have to do in this PR, just at some point

Reply:
You mean "useBiometry" if so I created a ticket with some issues I noticed along the way and this was one. I think it should be renamed to not use `use` in the name for clarity.